src/pages/quiz-page/QuizPage.jsx

```diff
@@ -2,6 +2,7 @@ import { useData } from "../../context/data-context";
 import { LeftArrow, RightArrow } from "../../assets/icons";
 import { useNavigate } from "react-router-dom";
 import { useEffect } from "react";
+import { useTheme } from "../../context/theme-context";
```

Comment:
suggestion: you have an index file in context folder. better to export from there to get a clean import here

Reply:
I have only two context, so I didn't create any index file for exporting.